src/core/components/libp2p.js

```diff
@@ -84,7 +86,7 @@ function defaultBundle ({ datastore, peerInfo, peerBook, options, config }) {
         }
       },
       EXPERIMENTAL: {
-        dht: get(options, 'EXPERIMENTAL.dht', false),
+        dht: !(get(options, 'local', false)),
```

Comment:
What does this mean?

Reply:
We have an option (`local` currently) that aims to:

> Run offline. Do not connect to the rest of the network but provide local API.

Regarding this, `go-ipfs` changed recently this option name to `offline`. So, I created a PR now for renaming this option to `offline` https://github.com/ipfs/js-ipfs/pull/1850. 

Once it is merged, I will rebase this PR and, if we go with `DHT` enabled by default in libp2p config, I will change this logic to not add the `dht` to the libp2p modules, if we are running offline.